### PR TITLE
Enable docker nvidia runtime on tegra devices

### PR DESCRIPTION
**Pull request by ricardosalveti**

### CODE REVIEW

**Comment by ricardosalveti** [comment]
`CORE_IMAGE_BASE_INSTALL:append = " tegra-libraries-core tegra-libraries-cuda"`

Then I'm able to run cuda containers:

`docker run -it --rm --net=host --runtime nvidia -e DISPLAY=$DISPLAY -v /tmp/.X11-unix/:/tmp/.X11-unix nvcr.io/nvidia/l4t-cuda:11.4.14-runtime`

**Review by MrCry0 — APPROVED** [review]

File: meta-lmp-base/recipes-containers/docker/docker-ce_%.bbappend [hunk 1]
```diff
@@ -14,13 +14,15 @@ SRC_URI:append = " \
 
 DOCKER_MAX_CONCURRENT_DOWNLOADS ?= "3"
 DOCKER_MAX_DOWNLOAD_ATTEMPTS ?= "5"
+DOCKER_DAEMON_JSON_CUSTOM ?= ""
```
Comment (quaresmajose) [inline meta-lmp-base/recipes-containers/docker/docker-ce_%.bbappend:17]: The addiction of the new DOCKER_DAEMON_JSON_CUSTOM variable can be skipped in more recent version of the `nvidia-container-toolkit` and the json snip can be installed to `${datadir}/containers/oci/hooks.d`.
AFAIK not supported yet on meta-tegra.

**Review by quaresmajose — APPROVED** [review]
LGTM

File: meta-lmp-base/recipes-containers/docker/docker-ce_%.bbappend (hunk 1 above)
Comment (ricardosalveti) [reply meta-lmp-base/recipes-containers/docker/docker-ce_%.bbappend:17]: Yeah, the one provided by the nvidia tegra bsp is quite behind the version used by x86. Tried to update some of the components but then got several issues.

**Pull request merged**